crates/hyperswitch_connectors/src/connectors/santander/transformers.rs

```diff
@@ -331,8 +662,103 @@ impl From<SantanderPaymentStatus> for common_enums::AttemptStatus {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
-pub struct SantanderPaymentsResponse {
+impl TryFrom<&str> for BrazilianState {
```

Comment:
for getting the state codes for Brazil, you don't need this util fn we already have a fn that exactly does the same - you can use it directly 
you can find it here - `to_state_code`
https://github.com/juspay/hyperswitch/blob/main/crates/hyperswitch_connectors/src/utils.rs#L1429 (please note - you have to add Brazil's alpha2 country code in this utility fn, rest everything else is already taken care of)